relentless/simulate/hoomd.py

```diff
@@ -428,7 +435,7 @@ def __call__(self, sim):
         """
         with sim.context:
             # setup FIRE minimization
-            fire = hoomd.md.integrate.mode_minimize_fire(dt=self.dt,
+            fire = hoomd.md.integrate.mode_minimize_fire(dt=self.options['max_displacement'],
```

Comment:
I can't tag it below, but line 447 is also like `max_evaluations` in that it runs 100 steps per iteration. Do you want to make this an option too? You could still use `max_evaluations`, and just document that it means the minimization runs for 100 steps each iteration.